app/views/facilities/index.html.haml

```diff
@@ -21,3 +21,19 @@
 
 - else
   .alert.alert-info= t(".empty", items: Facility.model_name.human(count: 2))
+
+- if @products.any?
+  %h4= text(".recently_purchased")
+  %table.table.table-striped.table-hover.product_list
+    %thead
+      %tr
+        %th
+    %tbody
+      - @products.each do |product|
+        %tr
+          %td
+            - if product.is_a?(Instrument)
+              = link_to product.name, [product.facility, product, :schedule]
```

Comment:
So this link location, and the link on line 38, are actually linking to the admin views for the product. We want to link to the ordering views (see the product list partial I linked above), since this feature is for regular users trying to buy things, not admins wanting to manage products.